test/app/controllers/users.test.js

```diff
@@ -0,0 +1,74 @@
+const request = require('supertest');
+const app = require('../../../app');
+
+describe('Create User', () => {
+  describe('With valid params', () => {
```

Comment:
Use describe to join test cases, for example : ' POST /users', 'Successful cases', 'Failure cases'